ironfist/src/cpp/combat/creatures.cpp

```diff
@@ -167,4 +205,51 @@ void UnloadCreatures() {
 		if(gArmyNames[i] != NULL) free(gArmyNames[i]);
 		if(gArmyNamesPlural[i] != NULL) free(gArmyNamesPlural[i]);
 	}
+}
+
+void __fastcall GetMonsterCost(int mon, int *const costs) {
+	
+	int i;
+	bool customed_sc = false;
+	for (i = 0; i < 6; ++i)
+	{
+		if (gMonSecondaryCost[mon][i][0])

```

Comment:
Once you get rid of customed_sc, this entire if-else will be equivalent to "costs[i] = gMonSecondaryCost[mon][i][1]".